flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/test/resources/pipeline-test.yaml

```diff
@@ -0,0 +1,31 @@
+################################################################################
+# Copyright 2023 Ververica Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+source:
+  type: mysql
+  name: MySQL Source
+  hostname: 172.29.0.237
+  port: 3306
+  username: root
+  password: AAbb0011
+  tables: atta_crm.\.*
+#  chunk-column: app_order_.*:id,web_order:product_id
+#  capture-new-tables: true
```

Comment:
We can release this feature in next release version, I'm worried we can not test well under current status